src/picongpu/include/simulation_defines/param/laserConfig.param

```diff
@@ -74,6 +74,13 @@ namespace picongpu
         /* laser phase shift (no shift: 0.0) */
         BOOST_CONSTEXPR_OR_CONST float_X LASER_PHASE = 0.0; /* unit: rad, periodic in 2*pi */
 
+        /* Use only the 0th Laguerremode for a standard Gaussian */
+        BOOST_CONSTEXPR_OR_CONST uint32_t MODENUMBER = 0;
+        PMACC_CONST_VECTOR(float_X, MODENUMBER + 1, LAGUERREMODES, 1.0);
+        /* This is just an example for a more complicated set of Laguerre modes. */
```

Comment:
So far we obtained our weights using Mathematica. The corresponding Python script takes forever and does not terminate. So have to figure out the various nonlinear optimization routines and its options in Python before putting such a script into `tools`.